packages/keystone/lib/Keystone/index.js

```diff
@@ -176,11 +179,14 @@ module.exports = class Keystone {
             itemId,
             itemIds,
           });
+        },
+        {
+          isPromise: true,
```

Comment:
I like this PR, could I make one minor style request that you put the `{ isPromise: true }` on one line rather than three in each case (unless `prettier` objects for some reason)? Thanks! 🙏

Reply:
👍